### PR TITLE
Expand on our file and resource storage solutions for client devs

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: _articles/mobile/files.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+The data APIs have additional APIs for workers and administrators (not discussed here). The data is not available to researchers or study coordinators. 
+
+## Participant Files API
+
+[Storing data files for a participant](/swagger-ui/index.html#/Participant%20Data) involves the following steps:
+
+1. Caller creates a [ParticipantFile](/model-browser.html#ParticipantFile) record through the [create participant file API.](/swagger-ui/index.html#/Participant%20Files/createParticipantFile) Please note that the mime/type for the file is required, although Bridge will currently allow you to create a file record without it;
+1. The returned ParticipantFile that is returned will have an `uploadURL` field with a presigned URL that can be used to `PUT` the file to S3;
+1. Client uploads the file to S3 (this call will fail if the file already exists);
```
Comment (DwayneJengSage) [inline _articles/mobile/files.md]: Two nitpicks:

1. It's not the call to S3 that fails, but the call to Bridge that fails.
2. This is considered a bug and we have a Jira to fix this https://sagebionetworks.jira.com/browse/BRIDGE-2926 Should we mention this restriction in the docs at all? And if we do, should we mention that the restriction is unintentionally and will be removed?

File: _includes/navigation.html [hunk 2]
```diff
@@ -53,7 +53,7 @@
             <a class="item" href="/articles/mobile/filtering.html">Customizing Content</a>
             <a class="item" href="/articles/mobile/authentication.html">Bridge Authentication</a>
             <a class="item" href="/articles/mobile/notifications.html">Notifications</a>
-            <a class="item" href="/articles/mobile/hosted_files.html">Hosted Files</a>
```
Comment (DwayneJengSage) [inline _includes/navigation.html:56]: If we're deleting this from the nav bar, are we also deleting it from our docs? If we're not deleting it from our docs, how are people going to access it?

**Review by DwayneJengSage — CHANGES_REQUESTED** [review]

File: _articles/mobile/files.md (hunk 1 above)
Comment (alxdarksage) [reply _articles/mobile/files.md]: Moved this and added an explanation that currently, the file cannot be overwritten, but we have plans to change this so the file can be updated.

File: _includes/navigation.html (hunk 2 above)
Comment (alxdarksage) [reply _includes/navigation.html:56]: All the hosted file information should now be in this files page. I can delete the old page (I was just keeping it for reference).

**Review by DwayneJengSage — APPROVED** [review]

**Pull request merged**